### DISCUSSION TITLE
Issue with GitHub Student Pack Application

### DISCUSSION THREAD

**Discussion by kamrullab** [community/community#142179] [GitHub Education]
Dear GitHub Education Team,

I am a student of Dhaka International University (DIU), and I am currently facing an issue while trying to apply for the GitHub Student Pack. Whenever I click on the application link:
 https://education.github.com/discount_requests/pack_application

I am automatically redirected to this page:
https://github.com/pricing

Additionally, after submitting a support ticket, the ticket does not appear in my dashboard. It shows the message "You don't have any tickets yet", even though I successfully submitted one. This is making it difficult for me to track the status of my request.

I am not sure why these issues are occurring, and it is preventing me from accessing the Student Pack benefits. Unfortunately, I do not have the financial means to purchase the paid plan right now. Therefore, I kindly request your assistance in resolving this issue, so I can proceed with my application.

Thank you for your support and assistance. I look forward to your prompt response.

Best regards,
Kamrul Hossain

![image.png](https://github.zendesk.com/attachments/token/8Wvs2o3REeFpxNRZLNW9JF2Ci/?name=image.png)

![image.png](https://github.zendesk.com/attachments/token/r5zfwJ6lyWFHRym6LFacesV9k/?name=image.png)

**Comment by kamrullab** [comment]
Dear GitHub Education Team,



Additionally, after submitting a support ticket, the ticket does not appear in my dashboard. It shows the message "You don't have any tickets yet", even though I successfully submitted one. This is making it difficult for me to track the status of my request.

I am not sure why these issues are occurring, and it is preventing me from accessing the Student Pack benefits. Unfortunately, I do not have the financial means to purchase the paid plan right now. Therefore, I kindly request your assistance in resolving this issue, so I can proceed with my application.

Thank you for your support and assistance. I look forward to your prompt response.

Best regards,
Kamrul Hossain


![image.png](https://github.zendesk.com/attachments/token/8Wvs2o3REeFpxNRZLNW9JF2Ci/?name=image.png)

![image.png](https://github.zendesk.com/attachments/token/r5zfwJ6lyWFHRym6LFacesV9k/?name=image.png)

**Comment by Aninooo** [comment]
Clear Browser Cache: Clear your browser's cache or try a different browser to avoid redirection problems.

Use a Different Device: If possible, try accessing the application link from another device.

Email Support: If the support ticket isn't visible, email GitHub support directly at support@github.com, explaining your situation and including any relevant details.

Check Eligibility: Ensure your university is on the list of eligible institutions for the GitHub Student Pack.
